web/src/EntryPage.js

```diff
@@ -94,6 +95,14 @@ class EntryPage extends React.Component {
         });
     };
 
+    if (this.state.application?.isRestricted) {
+      return Util.renderMessageLarge(this, `${i18next.t("check:Your IP address has been banned according to the configuration of application")} ${this.state.application.name}`);
```

Comment:
`has been banned according to the configuration of: %s"`

So we don't need three i18n texts